File: python-sdk/src/liveblocks/models/shared/activeusersresponse.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from liveblocks import utils


@dataclass_json
@dataclasses.dataclass
class ActiveUsersResponseData:
    connection_id: Optional[float] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('connectionId') }})
    type: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('type') }})
    

@dataclass_json
@dataclasses.dataclass
class ActiveUsersResponse:
    data: Optional[list[ActiveUsersResponseData]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('data') }})
    

File: typescript-sdk/src/sdk/models/shared/activeusersresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class ActiveUsersResponseData extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=connectionId" })
  connectionId?: number;

  @SpeakeasyMetadata({ data: "json, name=type" })
  type?: string;
}


export class ActiveUsersResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=data", elemType: ActiveUsersResponseData })
  data?: ActiveUsersResponseData[];
}
